.travis.yml

```diff
@@ -100,6 +101,11 @@ before_script:
 script:
   - if [[ $TRAVIS_OS_NAME == "linux" ]] && [[ $TRAVIS_RUST_VERSION == "stable" ]]; then
       rustup component add rustfmt && cargo fmt --all -- --check;
+      scripts/generate-bindings.sh && git diff HEAD;
```

Comment:
I don't think it will report error if the bindings is modified.

Reply:
I verified，see the log in travis

Reply:
Can you provide a link? As far as I can see, `git diff HEAD` will just display the diff and exit with success code 0.

Reply:
Maybe I made a mistake, I will verify and modify it tomorrow.